cmd/snap/main_test.go

```diff
@@ -80,6 +82,8 @@ func (s *BaseSnapSuite) RedirectClientToTestServer(handler func(http.ResponseWri
 
 type SnapSuite struct {
 	BaseSnapSuite
+
+	AuthFile string
```

Comment:
I'm sitting a bit on the fence here, it seems like AuthFile and the two helpers could go to the BuySnapSuite, OTOH it seems like a useful helper for other tests too. But then YAGNI. I will hear what the second reviewer thinks :)